kitsune/questions/views.py

```diff
@@ -455,6 +455,7 @@ def question_details(
         {
             "all_products": products,
             "all_topics": topics,
+            "all_tags": SumoTag.objects.filter(is_archived=False),
```

Comment:
Would you prefer to convert this to `SumoTag.objects.active()` now that https://github.com/mozilla/kitsune/pull/6383 has merged?